ext/opentelemetry-ext-sqlalchemy/src/opentelemetry/ext/sqlalchemy/__init__.py

```diff
@@ -0,0 +1,68 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Instrument `sqlalchemy`_ to report SQL queries.
+
+There are two options for instrumenting code. The first option is to use
+the `opentelemetry-auto-instrumentation` executable which will automatically
+instrument your SQLAlchemy engine. The second is to programmatically enable
+instrumentation via the following code:
+
+.. _sqlalchemy: https://pypi.org/project/sqlalchemy/
+Usage
+-----
+.. code:: python
+
+    from opentelemetry import trace
+    from opentelemetry.ext.sqlalchemy import SQLAlchemyInstrumentor
+    from opentelemetry.sdk.trace import TracerProvider
+    import sqlalchemy
+
+    trace.set_tracer_provider(TracerProvider())
+    trace_engine(engine, trace.get_tracer_provider())
```

Comment:
Shouldn't the recommended usage be `SQLAlchemyInstrumentor().instrument()`?

Reply:
Also if that's the case, is there still a need for `trace_engine()`?

Reply:
Good point, updated to remove the need for trace_engine.